<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>270</y>
          <width>113</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>320</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
       <widget class="QLabel" name="bienvenue">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>110</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Gestion du parking Bienvenue:</string>
        </property>
       </widget>
       <widget class="QLabel" name="connectezvous">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>240</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Connectez-vous:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton_Barriere1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Lever Barrière Entrée</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/parking.png</normaloff>../image pour qt/parking.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Baisser Barrière Entrée</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/barrier.png</normaloff>../image pour qt/barrier.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="camera1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>130</y>
          <width>281</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="camera6">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>140</y>
          <width>281</width>
          <height>201</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="camera3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>440</y>
          <width>291</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="camera7">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>450</y>
          <width>271</width>
          <height>211</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>190</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Lever Barrière Sortie</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/parking.png</normaloff>../image pour qt/parking.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>250</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Baisser Barrière Sortie</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/barrier.png</normaloff>../image pour qt/barrier.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Feu Vert</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/traffic-lights.png</normaloff>../image pour qt/traffic-lights.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Eclairage</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/light-bulb.png</normaloff>../image pour qt/light-bulb.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere6">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Feu Rouge</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/circle.png</normaloff>../image pour qt/circle.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>420</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Sirène</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/transistor-beacon-button.png</normaloff>../image pour qt/transistor-beacon-button.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Barriere9">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>370</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Ventillation</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/cooling.png</normaloff>../image pour qt/cooling.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>80</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
         </font>
        </property>
        <property name="text">
         <string>CAMERA-1</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
         </font>
        </property>
        <property name="text">
         <string>CAMERA-2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>390</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
         </font>
        </property>
        <property name="text">
         <string>CAMERA-3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>390</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>CAMERA-4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>380</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/icons8-cerclé-3-c-48.png</normaloff>../image pour qt/icons8-cerclé-3-c-48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>380</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/icons8-cerclé-4-c-48.png</normaloff>../image pour qt/icons8-cerclé-4-c-48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>80</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/icons8-cerclé-2-c-48.png</normaloff>../image pour qt/icons8-cerclé-2-c-48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>80</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../image pour qt/icons8-cerclé-1-c-48.png</normaloff>../image pour qt/icons8-cerclé-1-c-48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>30</y>
          <width>351</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rubik</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cliquer sur n'importe qu'elle chiffre pour mieux visualisé  </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="cam1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>861</width>
          <height>641</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="retour1">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>700</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>RETOUR</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="cam3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>781</width>
          <height>521</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="retour2">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>700</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>RETOUR</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QLabel" name="cam6">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>40</y>
          <width>731</width>
          <height>541</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="retour3">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>690</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>RETOUR</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_6">
       <widget class="QLabel" name="cam7">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>811</width>
          <height>581</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="retour4">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>690</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>RETOUR</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
